client/src/state/campaign/CampaignReducer.js

```diff
@@ -6,6 +6,7 @@ const initialState = deepFreeze({
   changeSetItemId: null,
   isPublishing: false,
   view: null,
+  newItemCreated: false,
```

Comment:
the name for this is quite confusing, since this is a boolean type, it should describe the question like `isNewItem` or `hasNewItem`.
`newItemCreated` implies it's an object that was just created.